.github/workflows/fevm-contract-tests.yaml

```diff
@@ -0,0 +1,150 @@
+        working-directory: fevm-contract-tests
+        run: |
+          npx hardhat --network local create-fund-accounts
+      - name: 'Run tests: web3.js SimpleCoin'
+        if: always()
+        env:
+          ROOT_PRIVATE_KEY: ${{ steps.testnode.outputs.ROOT_PRIVATE_KEY }}
+          USER_1_PRIVATE_KEY: ${{ steps.accounts.outputs.ACCOUNT1_PRIVATE_KEY }}
+        working-directory: fevm-contract-tests
+        run: |
+          npx hardhat --network local test ./test/web3.js/SimpleCoin.js
+      - name: 'Run tests: web3.js ERC20'
+        if: always()
+        timeout-minutes: 2
+        working-directory: fevm-contract-tests
+        env:
+          ROOT_PRIVATE_KEY: ${{ steps.testnode.outputs.ROOT_PRIVATE_KEY }}
+          USER_1_PRIVATE_KEY: ${{ steps.accounts.outputs.ACCOUNT1_PRIVATE_KEY }}
+        run: |
+          npx hardhat --network local test ./test/web3.js/ERC20.js
+      - name: 'Run tests: ethers.js SimpleCoin'
+        if: always()
+        timeout-minutes: 2
+        working-directory: fevm-contract-tests
+        env:
+          ROOT_PRIVATE_KEY: ${{ steps.testnode.outputs.ROOT_PRIVATE_KEY }}
+          USER_1_PRIVATE_KEY: ${{ steps.accounts.outputs.ACCOUNT1_PRIVATE_KEY }}
+        run: |
+          npx hardhat --network local test ./test/ethers.js/SimpleCoin.js
+      - name: 'Run tests: ethers.js ERC20'
+        if: always()
+        timeout-minutes: 2
+        working-directory: fevm-contract-tests
+        env:
+          ROOT_PRIVATE_KEY: ${{ steps.testnode.outputs.ROOT_PRIVATE_KEY }}
+          USER_1_PRIVATE_KEY: ${{ steps.accounts.outputs.ACCOUNT1_PRIVATE_KEY }}
+        run: |
+          npx hardhat --network local test ./test/ethers.js/ERC20.js
+      - name: 'Run tests: Uniswap -- SKIPPING'
+        if: always()
+        timeout-minutes: 200
+        working-directory: fevm-contract-tests
+        env:
+          DEPLOYER_PRIVATE_KEY: ${{ steps.testnode.outputs.ROOT_PRIVATE_KEY }}
+          USER_1_PRIVATE_KEY: ${{ steps.accounts.outputs.ACCOUNT2_PRIVATE_KEY }}
+          LOCAL_NODE_URL: 'http://127.0.0.1:8545'
+        run: |
+          echo skipping ; #cd ./extern/fevm-uniswap-v3-core && yarn install --frozen-lockfile && yarn compile &&  npx hardhat test test/UniswapV3Pool.spec.ts #  && yarn test
+      - name: 'Run tests: OpenZeppelin -- SKIPPING'
+        if: always()
+        working-directory: fevm-contract-tests
+        env:
+          USER_1_PRIVATE_KEY: ${{ steps.accounts.outputs.ACCOUNT1_PRIVATE_KEY }}
+          USER_2_PRIVATE_KEY: ${{ steps.accounts.outputs.ACCOUNT2_PRIVATE_KEY }}
+          USER_3_PRIVATE_KEY: ${{ steps.accounts.outputs.ACCOUNT3_PRIVATE_KEY }}
+          USER_4_PRIVATE_KEY: ${{ steps.accounts.outputs.ACCOUNT4_PRIVATE_KEY }}
+          USER_5_PRIVATE_KEY: ${{ steps.accounts.outputs.ACCOUNT5_PRIVATE_KEY }}
+        run: |
+          echo skipping ; #cd ./extern/openzeppelin-contracts/ && yarn && mkdir gasreport && npx hardhat --network itest test
```

Comment:
It would be great to have a comment to explain why this and the uniswap one are skipped, so we have an idea when and how to enable them. Maybe a ticket number?

Reply:
In the ongoing future now that we are storing all information on Linear we should be able to find their key information in the linked linear issues ENG-641 and ENG-642. For now however most discussion about openzeppelin and uniswap is on slack - https://filecoinproject.slack.com/archives/C04JR5R1UL8/p1706266000372379 is the most recent

Reply:
I have removed those sections from the yaml file 09f0d9bdbaa06fbe5a5f89d2a5662ca789876d82 with the comment `Removing uniswap and openzeppelin tests. Intention is to bring them in future prs. Documented in ENG-641 and ENG-642.`